WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -62,5 +62,33 @@ private static void StartOnWindowsStartup(bool runOnSystemStartup, string pathTo
 				key.DeleteValue(nameof(WalletWasabi), false);
 			}
 		}
+
+		private static async Task StartOnLinuxStartupAsync(bool runOnSystemStartup)
```

Comment:
There many linux distributions. Does this implementation work in all of them? I don't know, hence the question. Namely, does this work on Fedora?

Reply:
It was tested on Ubunu 18.04/20.04 and Debian 10 so far and worked on those. I'll test it on Fedora too.